overview.md

```diff
@@ -16,6 +16,8 @@ aliases: ['/docs/dev/key-features/','/tidb/dev/key-features','/docs/dev/overview
 
 [TiDB](https://github.com/pingcap/tidb) (/'taɪdiːbi:/, "Ti" stands for Titanium) is an open-source distributed SQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability. The goal of TiDB is to provide users with a one-stop database solution that covers OLTP (Online Transactional Processing), OLAP (Online Analytical Processing), and HTAP services. TiDB is suitable for various use cases that require high availability and strong consistency with large-scale data.
 
+TiDB Self-Managed is a product option of TiDB where users or organizations can manage and maintain the TiDB database system on their own infrastructure, whether in on-premise data centers, public clouds, or private clouds. TiDB Self-Managed gives you full control over the installation, configuration, scaling, and maintenance of TiDB clusters.
```

Comment:
To be updated.

Reply:
Updated in https://github.com/pingcap/docs/pull/18744/commits/98fc5d6aa18e51c8844b1c02249e9ab4542ccbce.